### ISSUE TITLE
Errors while starting

### ISSUE THREAD

**Issue by Zoidster** [binhex/arch-delugevpn#57]
Hi
I'm new to docker so please excuse any noob mistakes.
I am running docker on my Synology NAS. Running the docker command works fine, and I get a nice logfile.
The VPN connection starts flawlessly but the command crashes after trying to create folders in the mapped config folder. The user I am running the command with has read/write rights to the folder so I don't know why it would fail to create anything.

Log:
https://gist.github.com/Zoidster/8d96f29fe71b817e03e0a8d908b7756a
Run Command:
https://gist.github.com/Zoidster/8f3a3d5b1dda54ce4569b2a6afbf90ee

**Comment by Zoidster** [comment]
Nevermind, folder permissions were set wrong

**Comment by Zoidster** [comment]
Ok, now I have an exception in my log and I don't think thats permission-related

```
2018-03-28 23:26:20 | stdout | Exception: not a valid bencoded string
2018-03-28 23:26:20 | stdout | raise e
2018-03-28 23:26:20 | stdout | File "/usr/lib/python2.7/site-packages/deluge/ui/common.py", line 91, in __init__
2018-03-28 23:26:20 | stdout | torrent_info = uicommon.TorrentInfo(filename.strip(), 2)
2018-03-28 23:26:20 | stdout | File "/usr/lib/python2.7/site-packages/deluge/ui/web/json_api.py", line 714, in get_torrent_info
2018-03-28 23:26:20 | stdout | Traceback (most recent call last):
2018-03-28 23:26:20 | stdout | [ERROR   ] 01:26:20 json_api:717 not a valid bencoded string
2018-03-28 23:26:20 | stdout | 2018-03-29 01:26:20,275 DEBG 'deluge-web-script' stderr output:
```

**Comment by binhex** [comment]
due to you setting the permissions incorrectly its probably best to stop the container and delete everything in /config except the /config/openvpn folder, then start the container, this will then re-generate the files with the correct permissions.

**Comment by Zoidster** [comment]
I did that already and it fixed the write error from the original ticket.
The new error is what I got after everything else went smoothely.
The container is running just fine currently, and its in 24/7 use. The "not a valid bencoded string" error only occured in the initial startup.

After that there were no errors except for the following one, which occured three times about 7 days of usage.

```
2018-04-02 20:09:32,405 DEBG 'start-script' stderr output:
insmod: ERROR: could not load module /lib/modules/iptable_mangle.ko: No such file or directory
```
